src/main/java/net/glowstone/command/minecraft/EffectCommand.java

```diff
@@ -109,9 +109,9 @@ public boolean execute(CommandSender sender, String label, String[] args,
                     commandMessages.getResourceBundle());
             for (GlowPlayer player : players) {
                 player.addPotionEffect(
-                    new PotionEffect(effectType, duration, amplifier, false, hideParticles));
+                    new PotionEffect(effectType, duration, amplifier, false, !hideParticles));
                 doneMessage.send(sender, effectType.getName(),
-                        effectType.getId(), amplifier, player.getName(), duration / 20);
+                        effectType.getId(), amplifier + 1, player.getName(), duration / 20);
```

Comment:
This should just `amplifier` instead of `amplifier + 1`. The command is 0 to 255 inclusive, and potion effects start at 0. It is only the display on the client which is + 1, which does not need to be handled in the message.

Reply:
But in vanilla minecraft server this messages displays `amplifier + 1` and not from 0